composer.json

```diff
@@ -66,15 +66,11 @@
             "@lint",
             "@sniff",
             "@security",
-            "@test",
-            "@coverage"
+            "@test"
         ]
     },
     "config" : {
         "preferred-install": "dist",
-        "platform" : {
-            "php": "5.6.6"
```

Comment:
When we upgrade joindin-vm to PHP 7.3, I'd like to see this replaced with `"php": "7.3.1"` just so we don't get caught unawares when PHP 7.4 rolls around and we merge in a PR that requires 7.4.

Reply:
We'll actually already on 7.3, hence the twitter problem.